### PR TITLE
Tweak login-page + cookie notice

### PR DESCRIPTION
**Pull request by elwinschmitz**
See [AB#29353](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/29353)
See [AB#29321](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/29321)
See: [AB#29322](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/29322)

---

> [!TIP]
> Preview: https://lively-river-04adce503-5601.westeurope.5.azurestaticapps.net/

---

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

**Review by arsforza — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,3 @@
 {
-  "typescript.preferences.importModuleSpecifier": "non-relative",
-  "tailwindCSS.rootFontSize": 14
```
Comment (aberonni) [inline .vscode/settings.json:3]: IIRC this was necessary when opening the project from the root folder.

File: interfaces/Portalicious/.vscode/settings.json [hunk 2]
```diff
@@ -1,5 +1,12 @@
 {
   "typescript.preferences.importModuleSpecifier": "non-relative",
   "typescript.tsdk": "node_modules/typescript/lib",
-  "tailwindCSS.rootFontSize": 14
+  "prettier.configPath": ".prettierrc.yml",
+  "tailwindCSS.rootFontSize": 14,
+  "tailwindCSS.showPixelEquivalents": false,
```
Comment (aberonni) [inline interfaces/Portalicious/.vscode/settings.json:6]: Why?

File: interfaces/Portalicious/angular.json [hunk 3]
```diff
@@ -117,15 +117,15 @@
           "builder": "ng-extract-i18n-merge:ng-extract-i18n-merge",
           "options": {
             "buildTarget": "Portalicious:build",
-            "format": "xlf2",
+            "format": "xlf",
```
Comment (aberonni) [inline interfaces/Portalicious/angular.json:120]: Why?

File: interfaces/Portalicious/angular.json [hunk 4]
```diff
@@ -117,15 +117,15 @@
...
             "outputPath": "src/locale",
             "sourceFile": "messages.xlf",
             "targetFiles": ["messages.nl.xlf"],
             "newTranslationTargetsBlank": false,
             "sort": "stableAppendNew",
             "sortNestedTagAttributes": true,
             "trim": true,
-            "includeContext": true
+            "includeContext": false
```
Comment (aberonni) [inline interfaces/Portalicious/angular.json:128]: Why? I found this very useful.

**Review by aberonni — COMMENTED** [review]
Would be nice to have more reasoning in the PR description perhaps :)

**Pull request merged**